src/lib/components/utils/TagList.svelte

```diff
@@ -55,7 +55,9 @@
   const removeTag = (label: string) => {
     const idx = tags.findIndex((t) => t.name === label);
     tags = [...tags.slice(0, idx), ...tags.slice(idx + 1)];
-
+    if (tags.length == null) {
+      tags = [];
+    }
```

Comment:
The variable is set to an array right above this line, so the length can never be null

Reply:
I was seeing some strange behavior where it was getting set to null, this might not be happening anymore.